src/bin/ayb.rs

```diff
@@ -1,9 +1,9 @@
 use ayb::ayb_db::models::{DBType, EntityType};
+use ayb::formatting::TabularFormatter;
```

Comment:
I feel silly, bu where is this import used in the module?

Reply:
When calling `generate_table` and `generate_csv` :)

You need the trait to be in scope in order to call its functions. This is also the case for std traits, but most of them are added to the [prelude](https://doc.rust-lang.org/core/prelude/rust_2021/index.html) and are imported automagically.

Reply:
This explains so many questions! :)

(It also opens up so many more, but about Rust and not this PR :P)

Reply:
That's great! :)
Feel free to ask me privately any questions, I'll do my best to explain :p